conformance/README.md

```diff
@@ -0,0 +1,71 @@
+# Type System Conformance
+
+## Motivation
+
+[PEP 729](https://peps.python.org/pep-0729/) provides a structured and documented way to specify and evolve the Python type system. In support of this effort, an official [Python typing spec](https://github.com/python/typing/tree/main/docs/spec) has been drafted. This spec consolidates details from various historical typing-related PEPs. The spec will be modified over time to clarify unspecified and under-specified parts of the type system. It will also be extended to cover new features of the type system.
```

Comment:
Link to the rendered version: https://typing.readthedocs.io/en/latest/spec/index.html